adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/MDTagging.scala

```diff
@@ -71,8 +71,20 @@ private[read] case class MDTagging(
         contig <- Option(read.getContigName)
         if read.getReadMapped
       } yield {
-        maybeMDTagRead(read, referenceFileB.value
-          .extract(ReferenceRegion.unstranded(read)))
+        try {
+          maybeMDTagRead(read, referenceFileB.value
+            .extract(ReferenceRegion.unstranded(read)))
+        } catch {
+          case t: Throwable => {
+            if (validationStringency == ValidationStringency.STRICT) {
```

Comment:
The more we use `ValidationStringency` outside of forwarding to htsjdk, the less I like using their enum.

I know this isn't the place; what do you think of an ADAM-specific enum that maps to `ValidationStringency` where necessary?  We could use the opportunity to come up with a better name, since it isn't really about validation, it is more about error reporting.